<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-07-18T11:06:34Z" name="Suite" finished-at="2020-07-18T11:06:38Z" duration-ms="3811">
    <groups>
    </groups>
    <test started-at="2020-07-18T11:06:34Z" name="Test" finished-at="2020-07-18T11:06:38Z" duration-ms="3811">
      <class name="com.testDriver.TestDriver">
        <test-method is-config="true" signature="LoadingInitializer()[pri:0, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:34Z" name="LoadingInitializer" finished-at="2020-07-18T11:06:34Z" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadingInitializer -->
        <test-method signature="introTest()[pri:1, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:35Z" name="introTest" description="Intro" finished-at="2020-07-18T11:06:38Z" duration-ms="2216" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- introTest -->
        <test-method is-config="true" signature="LoadingInitializer()[pri:0, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:38Z" name="LoadingInitializer" finished-at="2020-07-18T11:06:38Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadingInitializer -->
        <test-method signature="testCase1()[pri:2, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:38Z" name="testCase1" description="First Testcase" finished-at="2020-07-18T11:06:38Z" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase1 -->
        <test-method is-config="true" signature="LoadingInitializer()[pri:0, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:38Z" name="LoadingInitializer" finished-at="2020-07-18T11:06:38Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadingInitializer -->
        <test-method signature="testCase2()[pri:3, instance:com.testDriver.TestDriver@212dfd39]" started-at="2020-07-18T11:06:38Z" name="testCase2" description="Second Testcase" finished-at="2020-07-18T11:06:38Z" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at com.mainbody.TestScenarios.secondTestCase(TestScenarios.java:25)
at com.testDriver.TestDriver.testCase2(TestDriver.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase2 -->
      </class> <!-- com.testDriver.TestDriver -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
